src/System Application/App/AI/src/Azure OpenAI/AzureOpenAIImpl.Codeunit.al

```diff
@@ -378,6 +378,7 @@ codeunit 7772 "Azure OpenAI Impl"
         end;
     end;
 
+    /// <returns>True if Function Call Response is set</returns>
```

Comment:
please remove this comment/doc. let's not start a bad trend of half baked doc sections... either you document the full thing or not... :-)

Reply:
Gone.

I think Darrick is planning to re-structure this bit future, so not gonna do the full document now.